packages/core/src/util/url.js

```diff
@@ -15,23 +15,24 @@ const secrets = require('../secrets');
 exports.sanitizeUrl = function sanitizeUrl(urlString) {
   let normalizedUrl;
   try {
-    // This currently uses the legacy URL API. As soon as we drop support for Node.js 6 we should move to the
-    // WHATWG URL API (https://nodejs.org/api/url.html#url_the_whatwg_url_api).
-    const p = url.parse(urlString);
-    if (p.protocol == null && p.host == null && p.pathname == null) {
+    const parsedUrl = new URL(urlString);
```

Comment:
Since The signature '(urlString: string): UrlWithStringQuery' of 'url.parse' is deprecated, USING https://nodejs.org/api/url.html#the-whatwg-url-api